buildSrc/src/main/java/org/elasticsearch/gradle/internal/InternalDistributionDownloadPlugin.java

```diff
@@ -149,24 +151,21 @@ private static String distributionProjectName(ElasticsearchDistribution distribu
             projectName += "no-jdk-";
         }
 
-        switch (distribution.getType()) {
-            case ARCHIVE:
-                return projectName + platform.toString() + archString + (platform == ElasticsearchDistribution.Platform.WINDOWS
-                    ? "-zip"
-                    : "-tar");
-
-            case DOCKER:
-                return projectName + "docker" + archString + "-export";
-
-            case DOCKER_UBI:
-                return projectName + "ubi-docker" + archString + "-export";
-
-            case DOCKER_IRON_BANK:
-                return projectName + "ironbank-docker" + archString + "-export";
-
-            default:
-                return projectName + distribution.getType();
+        if (distribution.getType() == ElasticsearchDistributionTypes.ARCHIVE) {
```

Comment:
Personally I'd have left this as a `switch` because I think it's clearer, but no big deal.

Reply:
Ah wait, it's not an `enum` any more, I see. Ignore me 😄